common/src/main/java/li/cil/scannable/common/scanning/ChestScannerModule.java

```diff
@@ -0,0 +1,78 @@
+package li.cil.scannable.common.scanning;
+
+import li.cil.scannable.api.scanning.BlockScannerModule;
+import li.cil.scannable.api.scanning.EntityScannerModule;
+import li.cil.scannable.api.scanning.ScanResultProvider;
+import li.cil.scannable.api.scanning.ScannerModule;
+import li.cil.scannable.client.scanning.ScanResultProviders;
+import li.cil.scannable.client.scanning.filter.BlockCacheScanFilter;
+import li.cil.scannable.client.scanning.filter.BlockScanFilter;
+import li.cil.scannable.client.scanning.filter.BlockTagScanFilter;
+import li.cil.scannable.client.scanning.filter.FriendlyEntityScanFilter;
+import li.cil.scannable.common.config.CommonConfig;
+import li.cil.scannable.common.config.Constants;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.state.BlockState;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+
+public enum ChestScannerModule implements BlockScannerModule {
+    INSTANCE;
+
+    private Predicate<BlockState> filter;
+
+    public static void clearCache() {
```

Comment:
I don't see this being called from the `ProviderCacheManagerImpl` classes?

Reply:
Ah, sorry, I was not clear enough here: I did not mean to the method can be removed, but that it should be called by these classes - as is the case for the modules that cache their filters, like the common/rare ores ones.